exercises/harry-potter/description.md

```diff
@@ -0,0 +1,57 @@
+This exercise is based on the exercise of the same name found at cyber-dojo.org
```

Comment:
This information is provided in the source section of the metadata and does not need to be repeated here.